quisp/modules/QRSA/HardwareMonitor/IHardwareMonitor.h

```diff
@@ -71,8 +71,10 @@ typedef std::map<int, InterfaceInfo> NeighborTable;
 // e.g.
 // "XX" -> {plus_plus = 56, plus_minus = 55, minus_plus = 50, minus_minus = 50},
 // "XY" -> {....
-typedef std::map<std::string, output_count> raw_data;
-typedef std::map<int, tomography_outcome> Temporal_Tomography_Output_Holder;  // measurement_count_id -> outcome. For single qnic
+typedef std::map<int, std::map<std::string, output_count>> raw_data;  // qnic -> partner . basis string . output
+typedef std::map<int, link_cost> link_cost_map;
+typedef std::map<int, tomography_outcome> TemporalTomographyOutputHolder;  // measurement_count_id -> outcome. For single qnic
```

Comment:
what does it mean `Holder` ? yes, all variables hold some value. should we declare it is `Holder`?  or any other meaning for the word?

Reply:
The `Holder` is nothing more than just holding the result. So it would be redundant.

Reply:
hmm, it's better than `TemporalTomograpyOutput`. if you have no idea, leave it now and later refactor it.

Reply:
Actually, this type is no longer used. So let me remove it in this PR.